src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -546,5 +555,28 @@ public void simulationPeriodic() {
     
     baseArmSim.update(Robot.kDefaultPeriod); // standard loop time of 20ms
     elbowArmSim.update(Robot.kDefaultPeriod);
+    
+    Logger.getInstance().recordOutput("Arm 2D Mechanism", simArmCanvas);
+    
+    //Absolute Zero
+    // Logger.getInstance().recordOutput("Base Arm Pose3D",  new Pose3d(0, 0, 0, new Rotation3d(0, 0, 0)));
+    // Logger.getInstance().recordOutput("Elbow Arm Pose3D", new Pose3d(0, 0, 0, new Rotation3d(0, 0, 0)));
+
+    //Just Offsets
+    // Logger.getInstance().recordOutput("Base Arm Pose3D",  new Pose3d(-0.238318, -0.004269, 0.177201, new Rotation3d(0, 0, 0)));
+    // Logger.getInstance().recordOutput("Elbow Arm Pose3D", new Pose3d(0.226824, -0.004269, 0.177202, new Rotation3d(0, 0, 0)));
+
+    //Offsets and Rotation
+    // Logger.getInstance().recordOutput("Base Arm Pose3D",  new Pose3d(-0.238318, -0.004269, 0.177201, new Rotation3d(0, -simBaseEncoderPosition, 0)));
+    // Logger.getInstance().recordOutput("Elbow Arm Pose3D", new Pose3d(0.226824, -0.004269, 0.177202, new Rotation3d(0, -simElbowEncoderPosition-(Math.PI/2), 0)));
+
+    //Offsets With Change
+    Logger.getInstance().recordOutput("Base Arm Pose3D",  new Pose3d(-0.238318, -0.004269, 0.177201, new Rotation3d(0, -simBaseEncoderPosition, 0)));
+    Logger.getInstance().recordOutput("Elbow Arm Pose3D", new Pose3d((0.226824*2.75)-armXPosition, -0.004269, 0.177202+armYPosition, new Rotation3d(0, -simElbowEncoderPosition-(Math.PI/2), 0)));
```

Comment:
This is how we determine the elbow angle in the Mechanism2d simulation. (Line 553-554)
```
    baseLigament.setAngle(Units.radiansToDegrees(baseArmSim.getAngleRads()));
    elbowLigament.setAngle(Units.radiansToDegrees(elbowArmSim.getAngleRads())-baseLigament.getAngle()-90);
```
You can copy this math in your Rotation3d or get them directly from the ligaments, e.g. elbowLigament.getAngle()